### PR TITLE
fix ignore /e2e/target folder

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### CODE REVIEW

**Comment by wu-sheng** [comment]
e2e fails?

**Comment by wu-sheng** [comment]
I rerun, but still fails. Strange. I can't see the issue from the logs too

File: e2e/Cargo.lock [hunk 1]
```diff
@@ -1,3 +1,5 @@
+# This file is automatically @generated by Cargo.
+# It is not intended for manual editing.
```
Comment (kezhenxu94) [inline e2e/Cargo.lock:2]: These lines are present in this file already, please either remove these or remove line 20-21

**Comment by kezhenxu94** [comment]
CI failure is not because of the expected data, but because of the response code (`response.status_code != 200`), check this,

https://github.com/apache/skywalking-rust/blob/a437859f5faa0aa13704a607a6ae6600228cd1ab/e2e/run_e2e.py#L38-L49

so problem might be there is queuing or delay in sending data to backend from the agent, just guess.

Also I think we should add retry for the validation as well despite of that my guess above is wrong

https://github.com/apache/skywalking-rust/blob/a437859f5faa0aa13704a607a6ae6600228cd1ab/e2e/run_e2e.py#L73

File: e2e/Cargo.lock (hunk 1 above)
Comment (tisonkun) [reply e2e/Cargo.lock:2]: OK. I'll regenerate the lock file and it should converge.

**Pull request merged**